railties/test/generators/migration_generator_test.rb

```diff
@@ -51,12 +51,12 @@ def test_add_migration_with_attributes
   end
 
   def test_add_migration_with_table_having_from_in_title
-    migration = "add_email_address_to_blacklisted_from_campaign"
+    migration = "add_email_address_to_restricted_list_from_campaign"
```

Comment:
Super minor because it's just a test string, but I think `excluded_from_campaigns` would keep the example clearer: I can't picture what goes in a table named `restricted_list_from_campaigns`.